### ISSUE TITLE
Error when renaming a field in Postgres table definition

### ISSUE THREAD

**Issue by 62mkv** [web2py/pydal#343]
Hi

We are trying to use pyDAL for "code-first" approach (for versioning of database schema).  
First we wrote 

```
db.define_table(
    'settings',
    Field('name', type='string', length=256, required=True),
    Field('admin_password', type='string', length=256, required=True),
    rname='dich.settings',
    migrate='dich.settings'
)
```

And a table was created OK. 

Then we've renamed "admin_password" field to "value": 

```
db.define_table(
    'settings',
    Field('name', type='string', length=256, required=True),
    Field('value', type='string', length=256, required=True),
    rname='dich.settings',
    migrate='dich.settings'
)
```

We've run the script and are getting Posgre error: relation "settings" does not exist.

in the sql.log, we see: 

> ALTER TABLE settings DROP COLUMN admin_password;

Apparently, pyDAL does not provide schema name for such operation. 

Is it our fault or a bug in pyDAL ? Thanks a lot !

**Comment by gi0baro** [comment]
@62mkv This seems a lack of schema name handling by pydal migrator, that uses just the name of the table. Will do some tests to check where we can change this.

**Comment by niphlod** [comment]
@gi0baro : this too should be targeted/inspected before the october-november release (i.e. not the next, but "second next" one)

**Comment by nextghost** [comment]
Will be fixed by PR #433
